src/actions/receive-profile.js

```diff
@@ -1142,7 +1249,7 @@ async function _extractProfileOrZipFromResponse(
         ),
       };
     default:
-      throw new Error(`Unhandled file type: ${(contentType: empty)}`);
+      throw new Error('Unhandled file type: ' + (contentType: empty));
```

Comment:
optional nit: nowadays we can use `throw assertExhaustiveCheck(contentType)`, I believe this was done before we had this thingie (or we just forgot)

Reply:
Ah, I'll make that change. I only touched this line because for some reason it was messing up the syntax highlighting in VS Code.